CMakeLists.txt

```diff
@@ -520,6 +520,9 @@ ELSE()
   ENDIF()
 ENDIF()
 
+# Portably get access to memset_s
+ADD_DEFINITIONS(-D__STDC_WANT_LIB_EXT1__=1)
```

Comment:
Modern Cmake best practices (https://cmake.org/cmake/help/latest/command/add_definitions.html) suggest using
more specific `ADD_COMPILE_DEFINITIONS(__STDC_WANT_LIB_EXT1__=1)`(notice that here we don't have `-D`) instead of `ADD_DEFINITIONS()`

Reply:
Sorry, available only in very modern CMAKE (starting from 3.12). Please, ignore the comment.